client/packages/common/src/ui/forms/JsonForms/jsonTemp/ui-schema.json

```diff
@@ -42,21 +42,48 @@
             },
             {
               "type": "Group",
-              "label": "Contact",
+              "label": "Contact Information",
               "elements": [
                 {
                   "type": "Control",
                   "scope": "#/definitions/Patient/properties/contactDetails",
-                  "keyField": "key",
-                  "options": {
-                    "detail": "GENERATED"
+                  "label": "Contacts",
+                  "itemLabel": "${index}: ${key}",
```

Comment:
Cool didn't know that this works

Reply:
I wrote my own little parser for it (nicked from Conforma) ;)